nav_msgs/Path path
---
string result